docs/hub/flair.md

```diff
@@ -0,0 +1,73 @@
+![alt text](https://raw.githubusercontent.com/flairNLP/flair/master/resources/docs/flair_logo_2020_FINAL_day_dpi72.png)
+
```

Comment:
```suggestion

```

Other libraries don't have a logo, so I wouldn't use one here.

Reply:
Paddle is one exception (https://huggingface.co/docs/hub/paddlenlp) but I can remove it here :hugs: